slave-nodejs6-ubuntu/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/fhwendy/jenkins-slave-nodejs-ubuntu:latest
+
+MAINTAINER Adam Saleh <asaleh@redhat.com>
+
+USER default
+
+RUN export NVM_DIR="$HOME/.nvm" && \
+    [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && \
+    nvm install v6.9.5 && \
+    nvm use v6.9.5 && npm i -g grunt && nvm alias default v6.9.5
```

Comment:
@AdamSaleh Might want to check what version of node 6 we are actually planning on targeting, i seen 6.9.1 being installed elsewhere.